Source/driver/Castro_io.cpp

```diff
@@ -1172,7 +1172,7 @@ Castro::plotFileOutput(const std::string& dir,
 #ifndef TRUE_SDC
     if (store_burn_weights) {
         MultiFab::Copy(plotMF, getLevel(level).burn_weights, 0, cnt, static_cast<int>(Castro::burn_weight_names.size()), 0);
-        cnt += static_cast<int>(Castro::burn_weight_names.size());
+        cnt += static_cast<int>(Castro::burn_weight_names.size());  // NOLINT
```

Comment:
It would be good to specify the check we're suppressing here: `// NOLINT(clang-analyzer-deadcode.DeadStores)`

Reply:
indeed, that's nice.

Reply:
updated